src/app/_shared/footer/footer.component.ts

```diff
@@ -1,9 +1,10 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-footer',
   templateUrl: './footer.component.html',
-  styleUrls: ['./footer.component.scss']
+  styleUrls: ['./footer.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
why do we have to do that?

Reply:
with Tailwindcss, it's recommended to disable ViewEncapsulation.
all styling now is using css classes from Tailwindcss, plus ViewEncapsulation.None is solve some issues if we want to do dark mode later.
in short, we don't need Angular ViewEncapsulation in these components.